lib/pf/detect/parser.pm

```diff
@@ -16,7 +16,15 @@ Base class for a pfdetect parser
 use strict;
 use warnings;
 use Moo;
+
+has id => (is => 'rw', required => 1);
+
+has path => (is => 'rw', required => 1);
+
+has type => (is => 'rw', required => 1);
  
+has enabled => (is => 'rw', default =>  sub { "enabled" });
```

Comment:
Same semantic logic as in pfmon, we should use status and not enabled as the attribute name